chapters/en/module3.md

```diff
@@ -12,19 +12,708 @@ id: 3
+
+- Use the data source `penguins` to make a **stacked** histogram plot.
+- Plot the counts of the `culmen_depth_mm`  and distinguish the penguin `species` using the color channel.
+- Make sure to set the `maxbins` argument to something appropriate. 
+- Give it an appropriate title. 
+
+
+<codeblock id="03_04">
+
+- Are you setting `alt.X('culmen_depth_mm', bin=alt.Bin(maxbins=30))`?
+- Are you setting a title in `properties()`?
+
+</codeblock>
+
+
+**Question**      
+
+Which species of penguin tend to have lower culmen depths?
+
+<choice id="1" >
+<opt text="Chinstrap">
+
+This species has a lot of overlap with the Adelie species. 
+
+</opt>
+
+<opt text="Gentoo" correct="true">
+
+These penguins tend to have a shorter culmen depth. 
+
+</opt>
+
+<opt text="Adelie" >
+
+This species has a lot of overlap with the Chinstrap species. 
+
+</opt>
+
+</choice>
 </exercise>
 
-<exercise id="3" title="Comparing multiple distributions" type="slides,video">
-<slides source="module3/module3_03" shot="1" start="0:003" end="07:12"> </slides>
+
+
+<exercise id="5" title="Layered plotting with Penguins">
+
+
+**Instructions:**    
+Be patient when running a coding exercise for the first time, it can take a few minutes. 
+
+**When you see `____` in a coding exercise, replace it with what you assume to be the correct code. Run the code to see if you obtain the desired output
+and submit it to validate if you were correct.**
+
+
+Using the [penguins](https://www.kaggle.com/parulpandey/palmer-archipelago-antarctica-penguin-data) dataset again, we are going to try to create a layered histogram. 
+
+
+<codeblock id="penguins">
+
+</codeblock>
+
+We want to understand how culmen depth differs over different penguin species. This means we will need to layer the histogram shapes of the `culmen_depth_mm` column. We also want to facet on the `sex` column to see if this affects the distribution. 
+
+Tasks: 
+
+Fill in the blanks in the code below so that the following gets accomplished.
+
+- Use the data source `penguins` to make a **layered** histogram plot and make sure to give it an appropriate opacity.
+- Plot the counts of the `culmen_depth_mm` and distinguish the penguin `species` using the color channel.
+- Make sure to set the `maxbins` argument to something appropriate and any other arguments needed to make a layered plot. 
+- Give it an appropriate title and facet by the `sex` column. 
+- Make sure to give each faceted plot an independent axis.
+
+<codeblock id="03_05">
+
+- Are you setting `opacity=0.4` in `mark_bar()`?
+- Are you setting `alt.X('culmen_depth_mm', bin=alt.Bin(maxbins=30))`?
+- Are you specifying `stack=None` in the `alt.Y()` helper function?
+- Are you setting a title in `properties()`?
+- Are you faceting by `sex` using `.facet('sex')`?
+- Are you giving the plots independent axis with `.resolve_scale(y='independent')`?
+
+</codeblock>
+
+**Question**      
+
+For each species of penguin, do the female penguins tend to have less deep culmen?
```

Comment:
This question would be A LOT easier to answer, if the two histograms in the question above were stacked vertically, instead of being side-by-side horizontally.

Reply:
Added columns=1 and adjusted the question and hint.